src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/TaintedDataAnalysis/SourceInfo.cs

```diff
@@ -64,7 +91,14 @@ public bool Equals(SourceInfo other)
                 && this.FullTypeName == other.FullTypeName
                 && this.IsInterface == other.IsInterface
                 && this.TaintedProperties == other.TaintedProperties
-                && this.TaintedMethods == other.TaintedMethods;
+                && this.TaintedMethodsNeedPointsToAnalysis == other.TaintedMethodsNeedPointsToAnalysis
+                && this.TaintedMethodsNeedsValueContentAnalysis == other.TaintedMethodsNeedsValueContentAnalysis
+                && this.TaintConstantArray == other.TaintConstantArray;
+        }
+
+        public bool RequiresValueContentAnalysis()
```

Comment:
>public bool RequiresValueContentAnalysis() [](start = 8, length = 42)

This could be a property, no?  Move it above with the other properties, and add /// <xml> comments.